src/components/pages/swap/Swap.jsx

```diff
@@ -393,6 +393,14 @@ const Swap = ({
                   0x1003d3574ac79303a5fa0951ecb04cc7acba9747) and it is the only official pBTC representation on BSC.
                 </InfoEta>
               ) : null}
+              {to && to.id === 'UOS' ? (
+                <InfoEta>
+                  By bridging to Ethereum you will get native ERC20 UOS tokens passing through the pUOS pToken
+                </InfoEta>
+              ) : null}
+              {to && to.id === PUOS_ON_ULTRA_MAINNET ? (
+                <InfoEta>By bridging to Ultra you will get native UOS tokens passing through the pUOS pToken</InfoEta>
```

Comment:
```suggestion
                <InfoEta>By bridging to Ultra you will get native UOS tokens passing through the pUOS pToken
</InfoEta>
```
Does it change something?

Reply:
The above item has the newline. That's why I'm asking

Reply:
Automatically managed by prettier/eslint

Reply:
The reason is prettier rule. Depending on the length of the line it removes or add new lines.
The message is rendered in the same way.